packaging/test_plugin_indexing.py

```diff
@@ -166,7 +166,13 @@ class BooksUnknownError(RuntimeError): pass
 
     return retvalue
 
-ES_VERSION = '7.x' # TODO We probably don't need this anymore.
+# By default, elasticsearch 8 enables TLS and requires auth credentials.
+# The iRODS consortium test hook captures the necessary values and exposes them
+# to the tests via environment variables. Without these, no communication with
+# elasticsearch 8 is possible.
+ES_AUTH_BASIC_CREDS = os.environ['IRODS_ES_AUTH_BASIC_CREDS']
+ES_PASSWORD = os.environ['IRODS_ES_PASSWORD']
+ES_TLS_CERT_FILE = os.environ['IRODS_ES_TLS_CERT_FILE']
```

Comment:
This is fine for now, but wanted to throw out an alternative for consideration...

Our tests in irods/irods use `test.settings` to capture things like this with `run_tests.py` setting the values based on inputs. Not saying that's better, but it is an alternative way to go about this.

Reply:
Cool. I'll write up an issue to investigate converting to `test.settings` after this PR is merged.

Reply:
I went ahead and made one, feel free to edit to your liking :) https://github.com/irods/irods_capability_indexing/issues/149